languages/swift/generic/Parse_swift_tree_sitter.ml

```diff
@@ -1736,12 +1745,18 @@ and map_arguments (env : env) ((v1, v2) : CST.interpolation_contents) :
   v1 @ v2
 
 and map_key_path_component (env : env) (x : CST.key_path_component) =
+  (* NOTE: According to a note at the callsite, this doesn't end up getting
+   * called in practice. At some point. *)
```

Comment:
Better to also add a comment there in case that ends up being modified? Probably okay as-is given return type being unit.

Reply:
There's already a comment there

Reply:
Was thinking more so about this function not really being implemented, beyond just that it isn't currently called since the list is always empty.

Reply:
I see, I can do that